UKHomeOffice/sgmr-service
# NMSW-114 & 115 Help content changes

### AC / Description of changes
 NMSW-114 - Change the link under 'Sailing pleasure craft to and from the UK' on /page/help to 'https://www.gov.uk/government/publications/sailing-a-leisure-craft-to-and-from-the-uk'

NMSW-115 - Make the below text at the bottom of the /page/help bold.
 'In exceptional circumstances...'

### To Test

- Run app locally
- Go to localhost:8080
- Sign in
- In the navigation click on 'help'
- Click on 'Sailing pleasure craft to and from the UK' link
- > It should take you to 'https://www.gov.uk/government/publications/sailing-a-leisure-craft-to-and-from-the-uk'
- Go back to the help page
- Scroll to the bottom
- > The last paragraph should be bold

### Notes

---

**Remember to merge to feature branches not `master`.**
